src/utils.ts

```diff
@@ -1198,3 +1198,19 @@ export function markdownToWebviewHtml(
   cleanHtml = cleanHtml.replace(/&lt;/g, "<").replace(/&gt;/g, ">");
   return cleanHtml;
 }
+
+export function isVersionGreaterOrEqual(baseVersion, targetVersion) {
```

Comment:
Can we verify if there is any duplicate of this util function `isVersionGreaterOrEqual` ? I think there are a few functions that does the same and maybe we could remove code duplication of this.

Reply:
You are right. Found a similar function.
I've updated the PR.